test/MinImports.lean

```diff
@@ -107,16 +117,26 @@ set_option linter.minImports true in
 set_option linter.minImports true
 
 /--
-warning: Imports increased to
+warning: Imports increased by 967 to
 [Mathlib.Tactic.Linter.MinImports]
+
+Now redundant: []
+
+New imports: [Mathlib.Tactic.Linter.MinImports]
+
 note: this linter can be disabled with `set_option linter.minImports false`
 -/
 #guard_msgs in
 #reset_min_imports
 
 /--
-warning: Imports increased to
+warning: Imports increased by 424 to
 [Mathlib.Tactic.FunProp.Attr, Mathlib.Tactic.NormNum.Basic]
+
+Now redundant: [Mathlib.Tactic.Linter.MinImports]
+
+New imports: [Mathlib.Tactic.FunProp.Attr, Mathlib.Tactic.NormNum.Basic]
+
```

Comment:
Also, swap the order. New is more interesting than redundant.

Reply:
Swapped the order.